cmake/Common.cmake

```diff
@@ -36,8 +41,12 @@ message(STATUS "Checking if doxygen/dot is installed:")
 message(STATUS " ")
 set(INSTALL_DOC_PATH ${CMAKE_INSTALL_PREFIX}/share/DGtal CACHE PATH "Installation directory for DGtal documentation files.")
 INCLUDE(doxygen)
-INCLUDE(TargetDoxygenDoc OPTIONAL)
-INCLUDE(TargetDoxygenDox OPTIONAL)
+if (MSVC)
+  INCLUDE(TargetDoxygenDox OPTIONAL)
+else (MSVC)
```

Comment:
Doc and dox are two (relevant) separate things. Why commenting the Doc one in MS ?

Reply:
For Visual Studio, we have to choose between Doc and Dox, because the target "winhelp" is referenced in "cmake/TargetDoxygenDoc.cmake" (line 98) and in "cmake/TargetDoxygenDox.cmake" (line 68), so there is an error during the cmake process if we keep the two "doc" and "dox". I made the choice of dox, but I could have chosen doc. Maybe renaming the target "winhelp" with, for example, "winhelpx" in "cmake/TargetDoxygenDox.cmake" would work ...

Reply:
I'd be in favor of the additional `winhelp-dox` target..

Reply:
Ok, I will correct this